common/include/wayland-gsdisplay.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
Why in common/? We won't reuse it outside of the displaying system, will we?

Reply:
It's referenced by `gui` as you mention and also once by `PAD` (since `PAD` has per-display-system input polling). It will also be used by GSdx once it's merged (the struct is currently duplicated into GSdx code).

Reply:
PAD is currently a part of the core as much as GUI is. common is used when libraries are used in plugins && core && utilities or any combination of it. Eventually this will be a part of the frontend part of the code but that's a bit far ahead, so you can move it into gui for now